build-scripts/config_common.cmake

```diff
@@ -162,6 +162,11 @@ if (WAMR_BUILD_LINUX_PERF EQUAL 1)
   endif ()
 endif ()
 
+if (NOT DEFINED WAMR_BUILD_SHRUNK_MEMORY)
```

Comment:
As we're adding a new build options, I think it'd be good to have a relevant documentation in one of the markdown files too. Could you add it?

Reply:
I've made updates. Please let me know if you have any further comments.